packages/replay/src/integration.ts

```diff
@@ -185,6 +185,17 @@ Sentry.init({ replaysOnErrorSampleRate: ${errorSampleRate} })`,
     this._replay.stop();
   }
 
+  /**
+   * Immediately send all pending events.
+   */
+  public flush(): Promise<void> | void {
+    if (!this._replay || !this._replay.isEnabled()) {
+      return;
+    }
+
+    return this._replay.flushImmediate();
```

Comment:
Wouldn't it better to only allow users to use debounced flush to reduce risk of being flooded with flushes?

Reply:
I'm good with that